pkg/config/analysis/analyzers/injection/injection-image.go

```diff
@@ -119,7 +119,7 @@ func (a *ImageAnalyzer) Analyze(c analysis.Context) {
 			return true
 		}
 
-		for _, container := range pod.Containers {
+		for _, container := range append(pod.Containers, pod.InitContainers...) {
```

Comment:
this will mutate `pod`, probably not what we want

Reply:
Fixed by using `slices.Clone`, thanks